explorer/client/src/utils/stringUtils.ts

```diff
@@ -90,6 +92,9 @@ export async function genFileTypeMsg(
         });
 }
 
+export const presentBN = (amount: BN) =>
```

Comment:
You could also use `Intl.NumberFormat` to accomplish this, which may work better if we ever support other languages / regions.

```typescript
new Intl.NumberFormat(undefined).format(amount);
```

Reply:
`format` expects a number while `amount` is of type `BN`. If we convert `BN` to `number`, we are limited to 53 bits - see here: https://github.com/indutny/bn.js/. Otherwise, this would be a great idea.

Reply:
If we had used native bigints, then the format method would work, but yeah makes sense given we use BN.